src/plugins/intel_npu/src/plugin/npuw/compiled_model.hpp

```diff
@@ -135,6 +136,8 @@ class CompiledModel : public ov::ICompiledModel {
     };
     std::vector<CompiledModelDesc> m_compiled_submodels;
 
+    bool m_update_required;
```

Comment:
it was per-tensor before. Why it is global now?

Reply:
It depends on `NPUW_FOLD` only - thus a single `bool`